mobile/src/main/java/com/alexstyl/specialdates/util/Notifier.java

```diff
@@ -89,19 +93,53 @@ public void forDailyReminder(ContactEvents events) {
                 );
 
         String title = NaturalLanguageUtils.joinContacts(context, events.getContacts(), 3);
-        String fullText = TextUtils.join(", ", events.getContacts());
 
         NotificationCompat.Builder builder = new NotificationCompat.Builder(context)
                 .setSmallIcon(R.drawable.ic_stat_contact_event)
                 .setContentTitle(title)
                 .setLargeIcon(largeIcon)
-                .setStyle(new NotificationCompat.BigTextStyle().bigText(fullText))
                 .setVisibility(NotificationCompat.VISIBILITY_PRIVATE)
                 .setAutoCancel(true)
-                .setContentText(fullText)
                 .setContentIntent(intent)
+                .setNumber(events.getContactCount())
```

Comment:
This is done right before displaying the notification (https://github.com/alexstyl/Memento-Namedays/pull/31/files#diff-a57d0b3fb13451a13012ab3380f979cfR155) so it is going to be overridden eventually. 

Can you remove the later one and keep yours as part of this PR please?